Prototypes/EtherstormWastes.cs

```diff
@@ -0,0 +1,170 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+using RBot;
+
+public class SepulchurePrequelAlden
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new CoreFarms();
+    public CoreAdvanced Adv = new CoreAdvanced();
+    public CoreStory Story = new CoreStory();
+    public CoreDailies Daily = new();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(3535))
+        {
+            
+            Core.Logger("You have already completed SelpulchurePrequel Storyline");
+            return;
+        }
+
+        Story.PreLoad();
+
+        Core.EquipClass(ClassType.Solo);
```

Comment:
not required per: 
![image](https://user-images.githubusercontent.com/47118168/165142270-b73cb274-65f0-4c94-9106-a0f8213a748b.png)